docs/reference/query-dsl/geo-shape-query.asciidoc

```diff
@@ -287,3 +287,10 @@ PUT /test/_doc/1
     }
 }
 --------------------------------------------------
+
+* {blank}
++
+include::{es-repo-dir}/mapping/types/geo-shape.asciidoc[tag=geoshape-dateline-orientation]
++
```

Comment:
Here we can just say the polygon orientation is assume to be RIGHT and point to how orientation works?